### ISSUE TITLE
Not utilizing GPU in training phase in goolgle colab

### ISSUE THREAD

**Issue by imtinan39** [matterport/Mask_RCNN#2102]
I am tring to traing color splash model of mask rcnn on colab. But unfortunately GPU is not utilizing. What should i do?
Tensorflow-1.5
Tensorflow-gpu-1.5
Keras-2.3.1 
Veraion currently using.

**Comment by matthewmcnulty** [comment]
Enable GPU in Edit -> Runtime Type

**Comment by matthewmcnulty** [comment]
Enable GPU in Edit -> Runtime Type

**Comment by imtinan39** [comment]
Tried but. Few moments later a notification pops out saying " you are connected to gpu but not utilizing it"

**Comment by susnato** [comment]
I think you should upgrade to Tensorflow 1.15.0 then try again.

**Comment by raditharch** [comment]
hello.. did you already solved it ? i think i got the same problem and solved it using 
!pip install tensorflow-gpu==1.9.0

but here.. come up with other problem, that one of module in tensorflow not found.

![image](https://user-images.githubusercontent.com/72731175/97666329-f8297e80-1aaf-11eb-9452-b0b80780dd7f.png)

Please, do help if any of you got the answer. Thanks.

**Comment by imtinan39** [comment]
> hello.. did you already solved it ? i think i got the same problem and solved it using
> !pip install tensorflow-gpu==1.9.0
> 
> but here.. come up with other problem, that one of module in tensorflow not found.
> 
> ![image](https://user-images.githubusercontent.com/72731175/97666329-f8297e80-1aaf-11eb-9452-b0b80780dd7f.png)
> 
> Please, do help if any of you got the answer. Thanks.

you can try using 
tensorflow-gpu=1.15.2
keras=2.1.5
hope that works
